### ISSUE TITLE
"latest" URL of DB-IP database

### ISSUE THREAD

**Issue by MaurUppi** [g-andrade/locus#38]
Alternative Providers
[DB-IP.com](https://db-ip.com/db/): lite databases seem to work but setting up auto-update for them is not practical, as there's no "latest" URL.

[Downloader for DB-IP.com Databases](https://github.com/MaurUppi/downloader)
develop by me. 

[dbip-asn-lite.csv.gz](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-asn-lite.csv.gz)
[dbip-asn-lite.csv.gz.sha256sum](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-asn-lite.csv.gz.sha256sum)
[dbip-asn-lite.mmdb.gz](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-asn-lite.mmdb.gz)
[dbip-asn-lite.mmdb.gz.sha256sum](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-asn-lite.mmdb.gz.sha256sum)
[dbip-city-lite.csv.gz](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-city-lite.csv.gz)
[dbip-city-lite.csv.gz.sha256sum](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-city-lite.csv.gz.sha256sum)
[dbip-city-lite.mmdb.gz](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-city-lite.mmdb.gz)
[dbip-city-lite.mmdb.gz.sha256sum](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-city-lite.mmdb.gz.sha256sum)
[dbip-country-lite.csv.gz](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-country-lite.csv.gz)
[dbip-country-lite.csv.gz.sha256sum](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-country-lite.csv.gz.sha256sum)
[dbip-country-lite.mmdb.gz](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-country-lite.mmdb.gz)
[dbip-country-lite.mmdb.gz.sha256sum](https://github.com/MaurUppi/downloader/releases/latest/download/dbip-country-lite.mmdb.gz.sha256sum)

**Comment by g-andrade** [comment]
Thanks for mentioning your project, it looks cool! I wonder if it goes against DB-IP's terms of service (or even GitHub's)? It would be important to clear that up before adding any mention of in `locus` - and even then, with the caveat that it's not an official source.

**Comment by MaurUppi** [comment]
> Thanks for mentioning your project, it looks cool! I wonder if it goes against DB-IP's terms of service (or even GitHub's)? 

I don't think it is either against DB-IP's terms of service or GitHub (what reason made you think it's relevant with them?)
the key is you MUST follow what they asked you to do according to DB-IP's terms of service. _copied below._

> It would be important to clear that up before adding any mention of in `locus` - and even then, with the caveat that it's not an official source.

Agreed with you about **it's not an official source.** in terms of the download link of the files. 



> The free IP to Country Lite database by [DB-IP](http://db-ip.com/) is licensed under a **[Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/)**

> You are free to use this IP to Country Lite database in your application, provided you give attribution to DB-IP.com for the data.

> In the case of a web application, **you must include a link back to DB-IP.com on pages that display or use results from the database.** You may do it by pasting the HTML code snippet below into your code :

> <a href='https://db-ip.com'>IP Geolocation by DB-IP</a>

**Comment by g-andrade** [comment]
> I don't think it is either against DB-IP's terms of service or GitHub (what reason made you think it's relevant with them?)

I don’t know friend, I assume commercial enterprises have their best interests at heart, no judgement about it - just practicality. Wouldn’t want to get DMCA’d two years from now for facilitating illegal distribution of intellectual property or something like that. `locus` is something I do for fun and I’d rather keep lawyers at bay.

That having been said (and I’m sorry if that came across in an unfriendly tone): I’ll review the relevant terms of service when I find the chance. Your project is ingenious and a clever way of improving the developer experience of making use of DB-IP databases. Thanks for pointing me to it!
